src/basic/signal-util.c

```diff
@@ -320,3 +321,15 @@ int parse_signo(const char *s, int *ret) {
 
         return 0;
 }
+
+void sigterm_handler(int signal, siginfo_t *info, void *ucontext) {
```

Comment:
If you move this maybe give it a more descriptive name? sigterm_process_group_handler?